README.md

```diff
@@ -145,6 +146,45 @@ The optional `revision` value is printed on the title page and in the header, if
 
 The manufacturer name, which must be specified in `system.yml` data document, is also show on the title page.
 
+## Audit Checklists
```

Comment:
Maybe this new section should go after the section "## RDM's Limitations"?

Reply:
BTW the section is great! Thanks for writing it up!

Reply:
I moved the section per your suggestion.